src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.5.0-beta.2
```

Comment:
This package is going to follow the convention of using the same major and minor version as its OpenTelemetry dependency Check this discussion: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1249/files#r1239021413

```suggestion
## 1.6.0-beta.1
```

Reply:
Makes sense